Drive/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include "./RoverDriveSystems.hpp"
+
+int main()
+{
+  string mode;
+  float speed;
+  units::angle::degree_t angle;
+
+  RoverDriveSystems roverDriver;
+  roverDriver.Initialize();
+
+      while (true)
+  {
+    // {speed, angle, mode } = getMissionControlCommands()
+
+    if (mode == "drive")
```

Comment:
Convert into a switch case and use only single character (D = drive, S = spin, T = translation)

Reply:
```C++
enum class DriveModes
{
  kDrive = 'D',
  kSpin = 'S',
  kTranslation = 'T',
}
```